bathlarp/frontend
# Feature/initial set up

**Issues affected**

No issues yet.

**Description**

Initial framework set-up.

**How to test**

Follow the install instructions and see if they work.
Use the package scripts to run the build and Storybook.

**Screenshots**

**Any other information?**

**Checklist**

- [x] Have all new components been added to Storybook?
- [x] Have unit/component/integration tests been written?
- [x] Has documentation been updated?
